src/Text/Markdown/SlamDown/Syntax/TextBox.purs

```diff
@@ -60,23 +62,23 @@ traverseTextBox eta = case _ of
   Time prec def → Time prec <$> eta def
   DateTime prec def → DateTime prec <$> eta def
 
-instance showTextBox ∷ (Functor f, Show (f String), Show (f HN.HugeNum), Show (f DT.Time), Show (f DT.Date), Show (f DT.DateTime)) ⇒ Show (TextBox f) where
+instance showTextBox ∷ (Functor f, Show (f String), Show (f D.Decimal), Show (f DT.Time), Show (f DT.Date), Show (f DT.DateTime)) ⇒ Show (TextBox f) where
   show = case _ of
     PlainText def → "(PlainText " <> show def <> ")"
     Numeric def → "(Numeric " <> show def <> ")"
     Date def → "(Date " <> show def <> ")"
     Time prec def → "(Time " <> show prec <> " " <> show def <> ")"
     DateTime prec def → "(DateTime " <> show prec <> " " <> show def <> ")"
 
-derive instance eqTextBox ∷ (Functor f, Eq (f String), Eq (f HN.HugeNum), Eq (f DT.Time), Eq (f DT.Date), Eq (f DT.DateTime)) ⇒ Eq (TextBox f)
-derive instance ordTextBox ∷ (Functor f, Ord (f String), Ord (f HN.HugeNum), Ord (f DT.Time), Ord (f DT.Date), Ord (f DT.DateTime)) ⇒ Ord (TextBox f)
+derive instance eqTextBox ∷ (Functor f, Eq (f String), Eq (f D.Decimal), Eq (f DT.Time), Eq (f DT.Date), Eq (f DT.DateTime), Eq1 f) ⇒ Eq (TextBox f)
+derive instance ordTextBox ∷ (Functor f, Ord (f String), Ord (f D.Decimal), Ord (f DT.Time), Ord (f DT.Date), Ord (f DT.DateTime), Ord1 f) ⇒ Ord (TextBox f)
```

Comment:
I don't understand this but I had to add `Eq1` and `Ord1` here to get this to type check.

I'm seeing this test failure locally:

```
Original:
   (Right (SlamDown ((Paragraph ((FormField "name" false (TextBox (PlainText (Compose (Just (Literal "Phil Freeman")))))) : Nil)) : Nil)))
Printed:
   (Right "name  = ______ (Phil Freeman)\n")
Parsed:
   (Right (SlamDown ((Paragraph ((FormField "name" false (TextBox (PlainText (Compose (Just (Literal "Phil Freeman")))))) : Nil)) : Nil)))
/Users/py/projects/slamdata/purescript-markdown/output/Data.Eq/index.js:13
var Eq1 = function (eq1) {
                   ^

RangeError: Maximum call stack size exceeded
    at new Eq1 (/Users/py/projects/slamdata/purescript-markdown/output/Data.Eq/index.js:13:20)
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:90:16
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:84:66
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:91:59
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:84:77
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:91:59
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:84:77
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:91:59
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:84:77
    at /Users/py/projects/slamdata/purescript-markdown/output/Data.Functor.Compose/index.js:91:59
* ERROR: Subcommand terminated with exit code 1
```

Reply:
Well, not a _test failure_ 😄